nolearn/lasagne/handlers.py

```diff
@@ -95,6 +95,51 @@ def __call__(self, nn, train_history):
             nn.save_params_to(path)
 
 
+class _RestoreBestWeights:
+    def __init__(self, remember):
+        self.remember = remember
+
+    def __call__(self, nn, train_history):
+        nn.load_params_from(self.remember.best_weights)
+        if self.remember.verbose:
+            print("Loaded best weights from epoch {} where {} was {}".format(
+                self.remember.best_weights_epoch,
+                self.remember.score or self.remember.loss,
+                self.remember.best_weights_loss,
+                ))
+
+
+class RememberBestWeights:
+    def __init__(self, loss='valid_loss', score=None, verbose=1):
+        self.loss = loss
+        self.score = score
+        self.verbose = 1
+        self.best_weights = None
+        self.best_weights_loss = None
```

Comment:
rather than calling this a loss in both the case of a score and a loss, perhaps negate it? That way comparison would also be the same.